spec/requests/admin/application_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "Admin::Applications", type: :request do
+  # include 'support/devise.rb'
+  describe '#authorized?' do
```

Comment:
don't write tests for private methods
leave this spec file empty for now

Reply:
left it empty/